_security/multi-tenancy/dynamic-config.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: default
+title: Dynamic configuration in OpenSearch Dashboards
+parent: OpenSearch Dashboards multi-tenancy
+nav_order: 147
+---
+
+
+# Dynamic configuration in OpenSearch Dashboards
+
+Dynamic configuration of multi-tenancy in OpenSearch Dashboards provides options to make common settings for tenancy without having to make changes to the configuration YAML files on each node and then restart the cluster. You can take advantage of this functionality by using the Dashboards interface or the REST API. The following list includes description of the options currently covered by dynamic configuration:
```

Comment:
Can we avoid making this into a feature by calling out the functionality? For example, "You can dynamically configure your multi-tenancy settings in OpenSearch Dashboards without making changes to the configuration YAML files on each note and then restarting the cluster."